Makefile

```
@@ -434,6 +434,14 @@ android-devices: export TARGET := android-sdk
 android-devices: ##@other Invoke adb devices
 	adb devices
 
+android-pull-geth: export VERSION ?= debug
+android-pull-geth:
+	adb pull "/storage/emulated/0/Android/data/im.status.ethereum$$( [ "$(VERSION)" = "release" ] || echo ".$(VERSION)" )/files/Download/geth.log"
+
+android-tail-geth: export VERSION ?= debug
```

By using a more exact smaller shell startup will be faster.
```suggestion
android-tail-geth: export TARGET := android-sdk
android-tail-geth: export VERSION ?= debug
```